jjb/paas/openshift-defaults.yml

```diff
@@ -0,0 +1,88 @@
+- defaults:
+    name: openshift-defaults
+    description: |
+        Maintainer: <a href="mailto:herlo@redhat.com>Clint Savage</a>
+        Secondary Maintainer: <a href="mailto:alivigni@redhat.com>Ari LiVigni</a>
+        Code Location: http://github.com/CentOS-PaaS-SIG/paas-sig-ci
+        IRC: #centos-devel on freenode<br/><br/>
+        Mailing List: <a href="mailto:centos-devel@centos.org">CentOS Devel</a>
+        Managed by Jenkins Job Builder. Do not edit via the web interface.
+    node: openshift-ansible-slave
+    wrappers:
+        - ansicolor
+        - workspace-cleanup
+        - timestamps
+        - inject:
+            properties-content: |
+                ANSIBLE_SSH_CONTROL_PATH=%(directory)s/%%h-%%r
+                ANSIBLE_HOST_KEY_CHECKING=False
+    properties:
+        - build-discarder:
+            num-to-keep: 50
+            artifact-num-to-keep: 15
+        - inject:
+            properties-content: |
+              lp_sha1={lpsha1}
+
+- scm:
+    name: paas-sig-ci-scm
+    scm:
+        - git:
+            url: 'git://github.com/herlo/paas-sig-ci.git'
+            branches:
+                - origin/master
+            basedir: paas-ci
+            skip-tag: true
+        - git:
+            url: 'git://github.com/CentOS-PaaS-SIG/duffy-ansible-module.git'
+            branches:
+                - origin/master
+            basedir: duffy-ansible-module
+            skip-tag: true
+        - git:
+            url: 'git://github.com/CentOS-PaaS-SIG/linch-pin.git'
+            branches:
+                - ${lp_sha1}
+            basedir: linch-pin
+            skip-tag: true
+
+- publisher:
+    name: openshift-email
+    publishers:
+      - email-ext:
+          recipients: herlo@redhat.com,tdawson@redhat.com
+          reply-to: $DEFAULT_REPLYTO
+          content-type: default
+          subject: $DEFAULT_SUBJECT CentOS-PaaS-SIG
+          body: $DEFAULT_CONTENT
+          attach-build-log: false
+          always: false
+          unstable: false
+          first-failure: true
+          not-built: false
+          aborted: true
+          regression: true
+          failure: false
+          improvement: false
+          still-failing: false
+          success: false
+          fixed: true
+          still-unstable: false
+          pre-build: false
+          matrix-trigger: only-configurations
+          send-to:
+            - requester
+            - recipients
+
+## Teardown Resources
```

Comment:
Is this something left over from my code? If so, i don't think we need it as we have a teardown publisher that actually works in paas-macros.yml